src/main/scala/Main.scala

```diff
@@ -88,27 +90,39 @@ object Main extends App {
         .map(c => c.sortKey)
     }
 
-    val wikiModel = new WikiModel("wiki/${image}", "wiki/${title}")
+    // not unused for now
+    // val wikiModel = new WikiModel("wiki/${image}", "wiki/${title}")
+
     val sortedEntries = labels.keys.toArray.sortWith(_.length > _.length)
-    val sentenceOption = SentenceOption("。", "", maxTokens)
+    val sentenceOption = SentenceOption("。", "", maxTokens, minTokens)
     val annotationOption = AnnotateOption("")
     val outputWriter = new FileWriter(new File("annotations.txt"))
+    val printEvery = 500
     var count = 0
+    var annotationList = List[String]()
 
     while (pageQueue.nonEmpty) {
       val dq = pageQueue.dequeue
       val link = dq._1
       pageQueue = dq._2
       val source = DbHelper.getPageSource(conn, link.pageId)
       if (source.nonEmpty) {
-        val text = wikiModel.render(new PlainTextConverter(), source.get)
```

Comment:
`PlainTextConverter` actually removes everything from the text source, including the link information we need.
For example: This useful piece of data was removed: `[[Text|Link]]`